README.md

```diff
@@ -93,6 +95,15 @@ Use the [App Manifest file](https://github.com/seratch/deepl-for-slack/blob/mast
 * Click **Open DeepL Tool**
 * Enter a text and the language on the modal
 
+### Note on auto translate feature
+* If activated the slack integration when added to a channel will automatically translate any incoming message.
+* It will not translate messages from other integrations.
+* As DeepL API does not support a target language based on DeepL's source language guess, it is unavoidable
+that some messages are translated twice (using up more cloud credits).
+* The bot can detect Japanese language based on the character set and `en,ja` will mostly not be translated twice.
+However `en,fr` will by default translate to English to French. If the guess was wrong, it will translate again.
```

Comment:
Maybe I was not clear on that. I has to translate twice only if the language was guessed wrongly.
With DeepL you cannot say translate to en->ja or ja->en, please decide, DeepL. But the source language comes back from the first translation attempt. If the guess was wrong it will use the source language DeepL decided on.
I can write this a little bit more clearly.